doc/Learn/Authentication/HowTo-MsalAuthentication.md

```diff
@@ -88,6 +88,39 @@ uid: Uno.Extensions.Authentication.HowToMsalAuthentication
     }
     ```
 
+    This configuration can also be done in the root App.cs file:
+
+    ```csharp
+    private IHost Host { get; set; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host =>
+            {
+                host
+                .UseAuthentication(builder =>
+                {
+                    builder.AddMsal(msal =>
+                        msal
+                        .Builder(msalBuilder => 
+                            msalBuilder.WithClientId("161a9fb5-3b16-487a-81a2-ac45dcc0ad3b"))
+                        .Scopes(new[] { "Tasks.Read", "User.Read", "Tasks.ReadWrite" })
+                    );
+                });
+            });
+        ...
+    }
+    ```
+
+    > [!WARNING]
+    > A ClientId of GUID format is required for MSAL Authentication to work. You can specify it in the appsettings.json file, or in the code itself.
+    > If the ClientId cannot be found, the following warning will be logged:
```

Comment:
This isn't correct - the application will crash with an exception. The details about the missing clientid are in the exception

Reply:
I've updated it to be the error instead

I didn't include the part about not being able to check for the ClientId on our side but I can put it in there if we should mention it as well